wallet/src/de/schildbach/wallet/rates/ExchangeRatesRepository.kt

```diff
@@ -109,7 +109,7 @@ class ExchangeRatesRepository private constructor() {
     }
 
     // This will return null if not found
-    fun getExchangeRate(currencyCode: String): ExchangeRate? {
+    fun getExchangeRate(currencyCode: String?): ExchangeRate? {
```

Comment:
it is possible for the currencyCode to be null and this was causing some crashes. 

This class was translated from Java and currencyCode was not specified as `@NotNullable`